PluginAutotask/API/Utility/Constants.cs

```diff
@@ -70,7 +119,7 @@ public static class Constants
             "TaskSecondaryResources",
             "TicketCategories",
             "TicketCharges",
-            "TicketHistory",
```

Comment:
Probably shouldn't be changing this as it will cause backwards compatibility problems with existing jobs using this schema.

Reply:
Never mind I see this is a constant and not actually changed.